packages/speech/src/index.js

```diff
@@ -55,7 +69,7 @@ function speechV1(options) {
 
   // Create the speech client with the provided options.
   var client = gapic.v1(options).speechClient(options);
-  Object.assign(client.constructor.prototype, helpers());
```

Comment:
`client.constructor.prototype` was adding the helper functions to the base Object prototype, i.e. every object created after this constructor is called will get the helper functions (odd). Since we return an instance of the client anyway I decided to assign the helper functions to the instance instead of the prototype. It didn't seem to have broken anything, but a second pair of eyes would be really appreciated

Reply:
Wow, that was indeed an oversight. Extending `client` is cool 👍